### ISSUE TITLE
upgrade networking.k8s.io/v1beta1 to networking.k8s.io/v1

### ISSUE THREAD

**Issue by softlion** [twuni/docker-registry.helm#19]
Message when installing:
```
networking.k8s.io/v1beta1 Ingress is deprecated in v1.19+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
```

k8s version: 1.21

**Comment by karezza** [comment]
Error: unable to build kubernetes objects from release manifest: unable to recognize "": no matches for kind "Ingress" in version "extensions/v1beta1"

**Comment by tarioch** [comment]
k8s 1.22 drops extensions/v1beta1 for ingress so this wont work out of the box anymore with it.

**Comment by muka** [comment]
PR #27  would make the helm ingress usable again in 1.22

**Comment by canterberry** [comment]
Released in [v2.0.0](https://github.com/twuni/docker-registry.helm/releases/tag/v2.0.0).

Because this changeset breaks compatibility with older Kubernetes versions, SemVer-wise I figure it's technically a major version increment.
